src/core/useMutableSource.ts

```diff
@@ -9,20 +9,28 @@ export {
 
 import { useEffect, useRef, useState } from 'react'
 
-const TARGET = Symbol()
-const GET_VERSION = Symbol()
+const TARGET = '_uMS_T'
```

Comment:
Why remove the use of `Symbol`?

Reply:
Because, with internal symbols, TS can't emit types properly. That's why `any` was used.